tests/helpers/has-attributes.js

```diff
@@ -9,20 +9,21 @@ import QUnit from 'qunit';
   @param {Array} expectedAttributes
 */
 function compareArrays(actualAttributes, expectedAttributes) {
-  let mapToArray = [];
-  actualAttributes.forEach(function(meta, name) {
-    mapToArray.push(name);
-  });
-  
   expectedAttributes.sort();
-  return mapToArray.sort().every(function(element, index) {
+  
+  return actualAttributes.sort().every(function(element, index) {
     return element === expectedAttributes[index]; 
   });
 }
 
 QUnit.assert.hasAttributes = function(actualAttributes, expectedAttributes) {
   this.expect(2);
 
-  this.ok(actualAttributes.size === expectedAttributes.length, `should have ${expectedAttributes.length} attributes`);
-  this.ok(compareArrays(actualAttributes, expectedAttributes), 'should have the expected attributes');
+  let mapToArray = [];
+  actualAttributes.forEach(function(meta, name) {
+    mapToArray.push(name);
+  });
```

Comment:
I would rename this `mapToArray` just `actualAttributesArray` or something. `mapToArray` sounds like a fn.